src/Microsoft.SqlTools.ServiceLayer/LanguageServices/LanguageService.cs

```diff
@@ -293,15 +295,22 @@ public void InitializeService(ServiceHost serviceHost, SqlToolsContext context)
             }
         }
 
-// turn off this code until needed (10/28/2016)
-#if false
-        private static async Task HandleDefinitionRequest(
-            TextDocumentPosition textDocumentPosition,
-            RequestContext<Location[]> requestContext)
+        internal static async Task HandleDefinitionRequest(TextDocumentPosition textDocumentPosition, RequestContext<Location[]> requestContext)
```

Comment:
Shouldn't we skip if intellisense is disabled just like for other request types like CompletionResolveRequest?